client/src/components/Login.jsx

```diff
@@ -90,19 +110,32 @@ class Login extends React.Component {
       return <Redirect to='/session' />;
```

Comment:
If we are getting rid of the button that fires `this.checkFormCompletion`, we should get rid of all the other functions that use it. This is Jorge's domain, so maybe this is worth doing another PR? Want to make sure we don't have zombie cruft that nobody wants to touch because nobody knows what it does.